tv.kodi.Kodi.yml

```diff
@@ -389,43 +531,39 @@ modules:
         --prefix=${FLATPAK_DEST} "pyyaml" --no-build-isolation
     sources:
       - type: file
-        url: https://files.pythonhosted.org/packages/36/2b/61d51a2c4f25ef062ae3f74576b01638bebad5e045f747ff12643df63844/PyYAML-6.0.tar.gz
-        sha512: b402993073282e7f4202823b051d364b91929362edd5b3aebe93b56833956ec9279c1ba82b97f8bc8a2b82d20e1060e4ec9fc90400a6ed902adce3e4f83a6e0e
+        url: https://files.pythonhosted.org/packages/54/ed/79a089b6be93607fa5cdaedf301d7dfb23af5f25c398d5ead2525b063e17/pyyaml-6.0.2.tar.gz
+        sha256: d584d9ec91ad65861cc08d42e834324ef890a082e591037abe114850ff7bbc3e
   - name: lirc
     no-autogen: true
     config-opts:
-      - "--with-systemdsystemunitdir=/app/lib/systemd"
+      - --with-systemdsystemunitdir=/app/lib/systemd
     sources:
       - type: archive
         url: https://downloads.sourceforge.net/project/lirc/LIRC/0.10.2/lirc-0.10.2.tar.bz2
         sha256: 3d44ec8274881cf262f160805641f0827ffcc20ade0d85e7e6f3b90e0d3d222a
+        x-checker-data:
+          type: anitya
+          project-id: 10762
+          url-template: https://downloads.sourceforge.net/project/lirc/LIRC/$version/lirc-$version.tar.bz2
       - type: shell
         commands:
           - sed -i s/Ubuntu/ignoreme/g configure.ac
       - type: patch
-        path: lirc-dont-build-docs.patch
-  - name: rapidjson
-    buildsystem: cmake-ninja
-    config-opts:
-      - "-DRAPIDJSON_BUILD_DOC=OFF"
-      - "-DRAPIDJSON_BUILD_EXAMPLES=OFF"
-      - "-DRAPIDJSON_BUILD_TESTS=OFF"
-    cleanup:
-      - /include
-      - /share/doc
-    sources:
-      - type: archive
-        url: "https://github.com/Tencent/rapidjson/archive/v1.1.0.tar.gz"
-        sha512: 2e82a4bddcd6c4669541f5945c2d240fb1b4fdd6e239200246d3dd50ce98733f0a4f6d3daa56f865d8c88779c036099c52a9ae85d47ad263686b68a88d832dff
```

Comment:
You are going to need to keep rapidjson for now. Several binary addons still use it. THe current failure in the build is inputstream.adaptive, however a bump to its version, will get it working, as rapidjson has been removed in https://github.com/xbmc/inputstream.adaptive/pull/1857

according to https://github.com/LibreELEC/LibreELEC.tv/pull/10009 the following pvr addons still use rapidjson, of which you are building them all still

pvr.zattoo
pvr.waipu
pvr.teleboy
pvr.plutotv

Reply:
readded